index.html

```diff
@@ -2,59 +2,223 @@
+        <a href="#" class="btn-text">Get my resume</a>*
+      <br> 
+      &nbsp;
+      </article>
+    </div>
+
+    <div class="achievement">
+      <div class="img">
+        <img src="./images/Desktop-4.png" alt="" srcset="">
+      </div>
+      <article>
+        <h3>Multi-Post Stories</h3>
+        <p class="infos">
+          <span>canopy</span> • Back End Dev • 2015
+        </p>
+        <p class="prg">A daily selection of privately personalized reads; no accounts or sign-ups required..</p>
+        <ul>
+          <li>HTML</li>
+          <li>XML</li>
+          <li>CSS</li>
+        </ul>
+        <a href="#" class="btn-text">Get my resume</a>*
+      <br> 
+      &nbsp;
+      </article>
+    </div>
+
+
+    <div class="achievement">
+      <div class="img">
+        <img src="./images/Snapshoot3.png" alt="" srcset="">
+      </div>
+      <article>
+        <h3>Tonic</h3>
+        <p class="infos">
+          <span>canopy</span> • Back End Dev • 2015
+        </p>
+        <p class="prg">A daily selection of privately personalized reads; no accounts or sign-ups required..</p>
+        <ul>
+          <li>HTML</li>
+          <li>XML</li>
+          <li>CSS</li>
+        </ul>
+        <a href="#" class="btn-text">Get my resume</a>*
+      <br> 
+      &nbsp;
+      </article>
+    </div>
+
+    <div class="achievement">
+      <div class="img">
+        <img src="./images/Snapshoot4.png" alt="" srcset="">
+      </div>
+      <article>
+        <h3>Multi-Post Stories</h3>
+        <p class="infos">
+          <span>canopy</span> • Back End Dev • 2015
+        </p>
+        <p class="prg">A daily selection of privately personalized reads; no accounts or sign-ups required..</p>
+        <ul>
+          <li>HTML</li>
+          <li>XML</li>
+          <li>CSS</li>
+        </ul>
+        <a href="#" class="btn-text">Get my resume</a>*
+      <br> 
+      &nbsp;
+      </article>
+    </div>
+
+  </section>
+  <section class="about-section">
+    <section class="about" id="About">
+
+      <h2 class="title">About <br> Myself</h2>
+
+      <p class="section-text">Hello I'm a software developer! I can help you build a product , feature or website Look
+        through some of my work and experience! If you like what you see and have a project you need coded, don't
+        hestiate to contact me.</p>
+
+      <p class="connect">Let's connect</p>
+
+      <div class="social-icon">
+        <ul>
+          <li><a href="https://twitter.com/"><img src="./images/twitter.png" alt="Twitter Icon"></a></li>
+          <li><a href="https://www.linkedin.com/in/"><img src="./images/linkedin.png" alt="Linkedin Icon"></a></li>
+          <li><a href="#"><img src="./images/medium.png" alt="Medium Icon"></a></li>
+          <li><a href="https://github.com/firstpj"><img src="./images/github.png" alt="GitHub Icon"></a></li>
+          <li><a href="#"><img src="./images/Fingers icon.png" alt="Fingers Icon"></a></li>
+        </ul>
+      </div>
```

Comment:
- [ ] Thumbs up for implementing this section 👍  but I noticed that the social icons are not well aligned, The first icon goes off to the edge of the screen. Please fix this as it will make your portfolio appear more professional. 😄 
- - Screenshot for reference:
![image](https://user-images.githubusercontent.com/107632780/229258552-0136720f-99e9-4b2f-9b23-5308b29b1ca5.png)